lib/logstash/inputs/jdbc.rb

```diff
@@ -160,6 +160,13 @@ class LogStash::Inputs::Jdbc < LogStash::Inputs::Base
   # Whether to force the lowercasing of identifier fields
   config :lowercase_column_names, :validate => :boolean, :default => true
 
+  # If not empty (default={}) it let's you select which string columns
+  # should be converted to UTF-8. To especify a column to be converted you can add the pair
+  # "column_name" => "original_encoding_type" to this hash. If you want to encode your columns
+  # you can use a definition like "_" => "original_encoding_type" and the encoding type will be
+  # applied to all columns.
+  config :columns_to_encode, :validate => :hash, :default => {}
```

Comment:
+1 on having:
 `default_encoding => nil` that executes no encoding change by default
`column_encoding => {}` for column specific encoding

so the rule would be:
  * if column_encoding exists for column, apply that encoding
  * else apply global_encoding if set. do nothing otherwise